testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+OK
+```
+where we are testing the class `TestCompressedXYZReader` which can be found in the module (file) `test\_coordinates.py`.
+
+
+### Running tests with setuptools ###
+
+Setuptools can also use [nose](http://somethingaboutorange.com/mrl/projects/nose) directly (and it takes care of having all the libraries in place):
+```
+python setup.py nosetests
+```
+
+If you have the [coverage](http://nedbatchelder.com/code/modules/rees-coverage.html) package installed, you can also check code coverage of the tests:
+```
+python setup.py nosetests --with-coverage --cover-package=MDAnalysis --cover-erase --cover-tests
+```
+
+# TODO: Fix further
+
+## Compatibility with numpy's test wrapper ##
+
+Up to MDAnalysisTests version 0.11.0 numpy's wrapper was used to run tests when invoked through `MDAnalysis.tests.test()` (but not through `nosetests`). This is now replaced by our own wrapper.
+
+Main differences are that numpy-specific arguments to `MDAnalysis.tests.test()` are now either emulated or simply not implemented. Here's a list of the ones most commonly used with MDAnalysisTests:
+
+* `label`: this allows the selection of tests based on whether or not they were decorated with the `@dec.slow` decorator. Current behavior recognizes only `label='fast'`. Any other label (or its absence) defaults to running all tests.
+* `verbose`: this argument is no longer accepted. Pass one of `-v`, `--verbose`, or `--verbosity=n` in `argv`;
+* `extra_argv`: this argument allows an extra list of flags to be passed to nose. It is still accepted, but deprecated in favor of nose's identical `argv` argument.
+
+Any other numpy-specific arguments will not be accepted and will cause the test run to fail.
+
+Under numpy the behavior was not to silence any test stdout output. The behavior now is to silence it; this can be reversed with the `-s` flag.
+
+## Data ##
+The simulation data used in tests are all released under the same license as MDAnalysis or are in the Public Domain (such as PDBs from the Protein Databank). An incomplete list of sources:
+* from Beckstein et al. (2009) (`adk.psf`,`adk_dims.dcd`)
+  * _adk\_dims_      Trajectory of a macromolecular transition of the enzyme adenylate kinase between a closed and an open conformation. The simulation was run in [CHARMM](http://www.charmm.org) c35a1.
+* unpublished simulations (O. Beckstein)
+  * _adk\_oplsaa_    Ten frames from the first 1 ns of a equilibrium trajectory of AdK in water with Na+ counter ions. The OPLS/AA forcefield is used with the TIP4P water model. The simulation was run with [Gromacs](http://www.gromacs.org) 4.0.2.
+* contributions from developers and users
+* Protein Databank
+
+## Purposely failing test ##
+
+It is sometime convenient to have a failing test. This is helpful, for instance, when testing a plugin for nosetests, or to make sure a script catches failure as expected. The `test_failure` test in `test_failure.py` can be made to fail by setting the `MDA_FAILURE_TEST` environment variable.
```

Comment:
I just read the test. It is to check that the open_file plugin works. Since the plugin is deleted now we should also delete the test. @utkbansal please open a PR to remove the corresponding test file.